DXMainClient/Domain/Multiplayer/CnCNet/TunnelHandler.cs

```diff
@@ -14,8 +14,20 @@ namespace DTAClient.Domain.Multiplayer.CnCNet
 {
     public class TunnelHandler : GameComponent
     {
+        /// <summary>
+        /// Determines the time between pinging the current tunnel (if it's set).
+        /// </summary>
         private const double CURRENT_TUNNEL_REFRESH_INTERVAL = 5.0;
```

Comment:
I think this interval is too short. Especially considering GameSurge's message delay of up to 3 seconds, this will cause issues with delayed commands and chat in the game lobby. Something like 20 seconds could be reasonable.